server/src/domain/media/media.service.ts

```diff
@@ -60,9 +60,9 @@ export class MediaService {
           size: JPEG_THUMBNAIL_SIZE,
           format: 'jpeg',
         });
-      } catch (error) {
+      } catch (error: any) {
         this.logger.warn(
-          `Failed to generate jpeg thumbnail using sharp, trying with exiftool-vendored (asset=${asset.id})`,
+          `Failed to generate jpeg thumbnail using sharp, trying with exiftool-vendored (asset=${asset.id}): ${error.message}`,
```

Comment:
Q for reviewers: Should we just remove this? vips/imagemagick should do this automatically. Might be nice to reduce code footprint and complexity. What do you think?

Reply:
This was a work around (fallback) for raw images that had an embedded thumbnail. If raw photos work without this I think we can definitely remove it.

Reply:
https://github.com/immich-app/immich/issues/2744